canonical/ubuntu.com
# Wd-10605 distributor shop step 1

## Done

- Build distributor shop step 1

## QA

- Go to /pro/distributor
- Initiate order
- Check it goes to "/pro/distributor/shop" and also check registration id is correct.

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-10605
